apps/dash-nlp/app.py

```diff
@@ -272,17 +376,27 @@ def plotly_wordcloud(df):
         ],
         "layout": {"height": "550", "margin": dict(t=20, b=20, l=100, r=20, pad=4)},
     }
-
-    return wordcloud_figure_data, frequency_figure_data
+    treemap_trace = go.Treemap(
+        labels=word_list_top,
+        parents=[""] * len(word_list_top),
+        values=list(map(lambda x: x * 10, freq_list_top)),
```

Comment:
@lundstrj  this is great 👍 .  Did you tried making this treemap without 10x?  The hover value on treemap is not consistent with those in wordcloud or in bar chart so it may be more clear to let them display same value. You may also try play with https://plot.ly/python/reference/#layout-yaxis-hoverformat in replacement of changing your values here.

Reply:
Let me know if it works smoothly then we are good to merge this

Reply:
I suppose it was a bit of a shortcut to use freq_list when I actually want the count of occurrences in the dataset (for that bank). Now when I tested it, I suppose the * 10 doesn't really add any value.

I'll remove it and consider using a different input.

Will look at the hoover format.